packages/components/src/DataTable/DataTable.css

```diff
@@ -0,0 +1,106 @@
+.dataTable {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+  border: 1px solid rgb(225, 225, 225);
```

Comment:
General comment/question: is there a reason we're not using Atlantis design tokens?

- [Borders](https://atlantis.getjobber.com/borders)
- [Color](https://atlantis.getjobber.com/colors)
- [Elevation](https://atlantis.getjobber.com/elevation)
- [Space](https://atlantis.getjobber.com/space)

etc

Reply:
We should have been using, this first draft was like a trial to see how tanstack, Atlantis, and the proposed design would fit together, we should have this following Atlantis' best practices.